.github/workflows/build.yaml

```diff
@@ -49,21 +53,54 @@ jobs:
       NeedRelease: ${{ env.NEED_RELEASE }}
 
   publish-release:
-    needs: [build]
+    needs: [build-docker-hub]
     runs-on: ubuntu-latest
-    if: ${{ needs.build.outputs.NeedRelease == 'true' }}
+    if: ${{ needs.build-docker-hub.outputs.NeedRelease == 'true' }}
     steps:
       - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
       - name: Generate deploy yaml
-        run: helm template deploy/chart --include-crds --set image.tag=${{ needs.build.outputs.ReleaseTag }} > deploy/everoute.yaml
+        run: helm template deploy/chart --include-crds --set image.tag=${{ needs.build-docker-hub.outputs.ReleaseTag }} > deploy/everoute.yaml
 
       - name: Create Release
         id: create_release
         uses: softprops/action-gh-release@v1
         with:
           files: deploy/everoute.yaml
-          name: ${{ needs.build.outputs.ReleaseTag }}
+          name: ${{ needs.build-docker-hub.outputs.ReleaseTag }}
+  
+  build-smtx-io:
+    runs-on: [self-hosted, pod]
+    steps:
+      - run: echo "IMAGE_TAG=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+
+      - if: ${{ github.ref == 'refs/heads/main' }}
+        run: echo "IMAGE_TAG=latest" >> $GITHUB_ENV
+
+      - run: echo "push image to everoute/release:${{ env.IMAGE_TAG }}"
+
+      - uses: actions/checkout@v2
+
+      - uses: docker/setup-qemu-action@v2
+
+      - run: docker context create build-smtx-io
+
+      - uses: docker/setup-buildx-action@v2
+        with:
+          endpoint: build-smtx-io
+
+      - uses: docker/login-action@v2
+        with:
+          registry: registry.smtx.io
+          username: ${{ secrets.HARBOR_PUSH_USERNAME }}
+          password: ${{ secrets.HARBOR_PUSH_TOKEN }}
+
+      - uses: docker/build-push-action@v2
+        with:
+          file: build/images/release/Dockerfile
+          tags: registry.smtx.io/everoute/release:${{ env.IMAGE_TAG }}
```

Comment:
could we build one time and push with 
tags: everoute/release:${{ env.IMAGE_TAG }}, registry.smtx.io/everoute/release:${{ env.IMAGE_TAG }}